hpx/parallel/algorithms/find.hpp

```diff
@@ -48,7 +48,7 @@ namespace hpx { namespace parallel { inline namespace v1
                 return std::find(first, last, val);
             }
 
-            template <typename ExPolicy, typename T>
+            template <typename ExPolicy, typename FwdIter, typename T>
```

Comment:
Isn't the template of function object supposed to be local_vector_iterator and that of the input and output of sequential and parallel local_raw_vector_iterator?